db/config.js

```diff
@@ -4,7 +4,66 @@ mongoose.connect('mongodb://localhost/test', {useNewUrlParser: true});
 
 const db = mongoose.connection;
 
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  // we're connected!
-});
\ No newline at end of file
+// db.collection("test").findOne({}, function(err, result) {
+//   if (err) throw err;
+//   console.log(result);
+//   db.close();
+// });
+
+
+
+
+
+const getAll = (callback) => {
+  db.collection("test").find({}).toArray((err, result) => {
+    if (err) {
+      callback(err)
+    } else {
+      callback(null, result)
```

Comment:
Queries like `find` in mongoose/mongodb return a [`Query`](https://mongoosejs.com/docs/queries.html#queries-are-not-promises) instance which is thenable like Promises. Just a heads up if you don't like using the callback pattern.